src/main/java/com/smartcar/sdk/Vehicle.java

```diff
@@ -126,6 +130,48 @@ public String vin() throws SmartcarException {
     return this.call("permissions", "GET", null, ApplicationPermissions.class).getData().getPermissions();
   }
 
+  /**
+   * Checks if permissions granted to a vehicle contain the specified permission.
+   *
+   * @param permission Check if the vehicle has this permission granted
+   *
+   * @return Whether the vehicle has the specified permission
+   * @throws SmartcarException if the request is unsuccessful
+   */
+  public boolean hasPermissions(String permission) throws SmartcarException {
+    try {
+      List<String> vehiclePermissions = Arrays.asList(this.permissions());
+      permission = permission.replaceFirst("^required:", "");
+
+      return vehiclePermissions.contains(permission);
+    } catch (SmartcarException exception) {
+      throw exception;
+    }
+  }
+
+  /**
+   * Checks if permissions granted to a vehicle contain the specified permissions.
+   *
+   * @param permissions Check if the vehicle has these permissions granted
```

Comment:
Same as above

Reply:
Sydney and I discussed it and thought this was the best description for the parameter, it is the same description in all SDKs.